proto/cosmos/crypto/ed25519/keys.proto

```diff
@@ -19,5 +32,8 @@ message PubKey {
 // Deprecated: PrivKey defines a ed25519 private key.
 // NOTE: ed25519 keys must not be used in SDK apps except in a tendermint validator context.
 message PrivKey {
+  option (legacy_amino.name)     = "tendermint/PrivKeyEd25519";
+  option (legacy_amino.encoding) = "bytes field";
```

Comment:
what does it do to have `(legacy_amino.encoding)` on a `message` level?

Reply:
same question as https://github.com/cosmos/cosmos-sdk/pull/13501#discussion_r1006108527